classes/Homepage.php

```diff
@@ -31,7 +31,10 @@ public function display() {
         $data = array();
 
         $data['debates'] = $this->getDebatesData();
-        $data['mp_data'] = $this->getMP();
+
+        $user = new User();
+        $data['mp_data'] = $user->getMP($this->mp_url, $this->cons_type, $this->mp_house);
```

Comment:
First argument of this shouldn't be here any more, after 3db486c8e8ba2c43c5ffaa29e8dca58be9438582